ui/src/containers/NodeCreateForm.tsx

```diff
@@ -186,46 +190,47 @@ const NodeCreateForm = () => {
-                error={
-                  !(
-                    values.workload_plane ||
-                    values.control_plane ||
-                    values.infra
-                  )
-                    ? intl.formatMessage({
-                        id: 'role_values_error',
-                      })
-                    : undefined
-                }
-                content={
-                  <CheckboxGroup>
-                    <Checkbox
-                      name="workload_plane"
-                      label={intl.formatMessage({
-                        id: 'workload_plane',
-                      })}
-                      checked={values.workload_plane}
-                      value={values.workload_plane}
-                      onChange={handleChange('workload_plane')}
-                      onBlur={handleOnBlur}
-                    />
-                    <Checkbox
-                      name="control_plane"
-                      label={intl.formatMessage({
-                        id: 'control_plane',
-                      })}
-                      checked={values.control_plane}
-                      value={values.control_plane}
-                      onChange={handleChange('control_plane')}
-                      onBlur={handleOnBlur}
-                    />
-                    {isPureMetalK8s && (
+              {isMetalK8s ? (
+                <FormGroup
+                  label={intl.formatMessage({
+                    id: 'roles',
+                  })}
+                  id="roles"
+                  helpErrorPosition="bottom"
+                  error={
+                    !(
+                      values.workload_plane ||
+                      values.control_plane ||
+                      values.infra
+                    )
+                      ? intl.formatMessage({
+                          id: 'role_values_error',
+                        })
+                      : undefined
+                  }
+                  content={
+                    <CheckboxGroup>
+                      <Checkbox
+                        name="workload_plane"
+                        label={intl.formatMessage({
+                          id: 'workload_plane',
+                        })}
+                        checked={values.workload_plane}
+                        value={values.workload_plane}
+                        onChange={handleChange('workload_plane')}
+                        onBlur={handleOnBlur}
+                      />
+                      <Checkbox
+                        name="control_plane"
+                        label={intl.formatMessage({
+                          id: 'control_plane',
+                        })}
+                        checked={values.control_plane}
+                        value={values.control_plane}
+                        onChange={handleChange('control_plane')}
+                        onBlur={handleOnBlur}
+                      />
+
                       <Checkbox
```

Comment:
Actually we'd like to hide only the infra roles for artesca instance. We should still keep the Roles of `Workload Plane` and `Control Plane`.

![image](https://github.com/scality/metalk8s/assets/18453133/4e3b4c94-7d7d-4846-949c-158cd1e0bd08)